yabox/algorithms/base.py

```diff
@@ -10,23 +10,21 @@ def binomial_crossover(target, mutant, cr):
     return np.where(p, mutant, target)
 
 
-def random_sample(population, exclude, size):
+def random_sample(population, exclude, size=3):
     # Optimized version using numpy
-    idxs = [i for i in range(population.shape[0]) if i != exclude]
```

Comment:
I need to check the performance of using np.random.choice because I've used it in the past and I think for performance reasons I decided to keep the random.shuffle strategy

Reply:
No problem, if I have time I'll check it and post results